src/OctoshiftCLI.Tests/ado2gh/Commands/ShareServiceConnectionCommandTests.cs

```diff
@@ -37,11 +37,47 @@ public void Should_Have_Options()
         [Fact]
         public async Task Happy_Path()
         {
+            var _mockAdoApi = new Mock<AdoApi>(_mockAdoClient.Object, null, null) { CallBase = true };
+
             var adoOrg = "FooOrg";
             var adoTeamProject = "BlahTeamProject";
             var serviceConnectionId = Guid.NewGuid().ToString();
             var teamProjectId = Guid.NewGuid().ToString();
+            var serviceConnection = @$"
+            {{
+                ""data"": {{}},
+                ""id"": ""{serviceConnectionId}"",
+                ""name"": ""MyNewServiceEndpoint"",
+                ""type"": ""Generic"",
+                ""url"": ""https://myserver"",
+                ""createdBy"": {{
+                    ""displayName"": ""Chuck Reinhart"",
+                    ""url"": ""https://vssps.dev.azure.com/fabrikam/_apis/Identities/e18a1f0a-b112-67fd-a9e0-e3bb081da49e"",
+                    ""_links"": {{
+                        ""avatar"": {{
+                            ""href"": ""https://dev.azure.com/fabrikam/_apis/GraphProfile/MemberAvatars/msa.ZTE4YTFmMGEtYjExMi03N2ZkLWE5ZTAtZTNiYjA4MWRhNDll""
+                        }}
+                    }},
+                  ""id"": ""e18a1f0a-b112-67fd-a9e0-e3bb081da49e"",
+                  ""uniqueName"": ""fabfiber@outlook.com"",
+                  ""imageUrl"": ""https://dev.azure.com/fabrikam/_apis/GraphProfile/MemberAvatars/msa.ZTE4YTFmMGEtYjExMi03N2ZkLWE5ZTAtZTNiYjA4MWRhNDll"",
+                  ""descriptor"": """"
+                }},
+                ""description"": """",
+                ""authorization"": {{
+                    ""parameters"": {{
+                        ""username"": ""myusername"",
+                        ""password"": null
+                    }},
+                    ""scheme"": ""UsernamePassword""
+                }},
+                ""isShared"": false,
+                ""isReady"": true,
+                ""owner"": ""Library"",
+                ""serviceEndpointProjectReferences"": []
+              }}";
```

Comment:
As I explained, this need to go into `AdoApiTests`. All we need in here is to properly setup the `AdoApi#ContainsServiceConnection` method.